exporter/redis.go

```diff
@@ -370,15 +370,26 @@ func parseConnectedSlaveString(slaveName string, slaveInfo string) (offset float
 	return
 }
 
-func extractConfigMetrics(config []string, addr string, alias string, scrapes chan<- scrapeResult) error {
+func extractConfigMetrics(config []string, addr string, alias string, scrapes chan<- scrapeResult) (dataBasesCount int, err error) {
 	if len(config)%2 != 0 {
-		return fmt.Errorf("invalid config: %#v", config)
+		err = fmt.Errorf("invalid config: %#v", config)
+		return
```

Comment:
can you use `return 0, fmt.Errorf....` - makes it more explicit what's returned.